SU2_CFD/src/drivers/CDiscAdjMultizoneDriver.cpp

```diff
@@ -421,6 +421,8 @@ void CDiscAdjMultizoneDriver::Run() {
         }
       }
 
+      // TobiKattmann:: Add dual time terms dG/du from n+1 and N+2 to external
```

Comment:
I think there is an open PR for that kind of stuff :)

Reply:
Yes, and i sure need to take a more in depth look at the PR. This is leftover of my own try to think that trough and will be removed

Reply:
I think your thinking of it is very good, and aligned with Ole's idea with the external that combines the OF gradient with MZ cross terms.
All that would be needed is for `CDiscAdjSolver::SetAdjoint_Output` to not add the dual time contribution while setting the adjoints.

Reply:
Yes, but on the other hand one could keep the same logic as for single zone to some extent, which  would be nice

Reply:
That would not work with GMRES applied to the adjoint, and that is something we want :)